modules/update-restricted-network-cli.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network.adoc
+
+[id="update-restricted-network-cli_{context}"]
+= Update the restricted network cluster
+
+Verify the sig
+
+.Prerequisites
+
+* You have access to the mirror registry that you used to store the images that
+you used to install {product-title}.
+* You have the `ImageContentSourcePolicy` information for the
+* The the OpenShift Command-line Interface (CLI), commonly known as `oc`, is
+installed.
+* The `jq` package is installed.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Create an `ImageContentSourcePolicy` resource for your cluster:
```

Comment:
You should already have an ImageContentSourcePolicy.  If you mirror from the same repos to the same mirror repo, you won't need to update it.  Basically, folks should run `oc adm release mirror ...` and then merge the `ImageContentSourcePolicy` it suggests with the values that are already in their cluster, and only creating a fresh ImageContentSourcePolicy if none already exists.

Reply:
@wking, when is the ImageContentSourcePolicy created? I don't see an explicit step to do it. Is the policy created when you drop the imageContentSources data into your `install-config.yaml` during installation?

If you can share the command to edit the existing policy, I'm happy to document that instead.

Reply:
> when is the ImageContentSourcePolicy created?

The installer creates it if you set `imageContentSources`.  Otherwise, you'd have to create it from scratch.